assemblies/enabling-ai-safety-with-guardrails.adoc

```diff
@@ -0,0 +1,52 @@
+:_module-type: ASSEMBLY
+
+ifdef::context[:parent-context: {context}]
+[id="enabling-ai-safety-with-guardrails_{context}"]
+= Enabling AI safety with Guardrails
+
+The TrustyAI Guardrails Orchestrator service is a tool to invoke detections on text generation inputs and outputs, as well as standalone detections.
+
+It is underpinned by the open-source project link:https://github.com/foundation-model-stack/fms-guardrails-orchestrator[FMS-Guardrails Orchestrator] from IBM. You can deploy the Guardrails Orchestrator service through a Custom Resource Definition (CRD) that is managed by the TrustyAI Operator.
+
+The following sections describe the Guardrails components, how to deploy them and provide example use cases of how to protect your AI applications using these tools:
+
+Detectors::
+Explore the available detector types in the Guardrails framework. Currently supported detectors are:
+    - The built-in detector: Out-of-the-box guardrailing algorithms for quick setup and easy experimentation
```

Comment:
needs final period "." - we probably should be consistent with period usage throughout this section